bin/proving-service/Cargo.toml

```diff
@@ -82,5 +82,5 @@ miden-testing = { workspace = true, default-features = false }
 miette = { version = "7.5", features = ["fancy"] }
 prost = { version = "0.13", default-features = false, features = ["derive"] }
 prost-build = { version = "0.13" }
-protox = { version = "0.7" }
+protox = { version = "0.8" }
 tonic-build = { version = "0.12" }
```

Comment:
Not for this PR: but I think there are now 0.13 versions of tonic crates now available. We should probably update these in subsequent PRs.

Reply:
Created #1384